docs/project-setup/existing-projects.md

```diff
@@ -1,3 +1,32 @@
 # Existing Projects
 
-To Be Done
\ No newline at end of file
+Outrigger is intended to work with either raw Docker commands or Docker Compose files. Docker
+Compose is the preferred tool to use when setting up projects and in this section we will go
+over some basics to get a project setup.
+
+Be sure to read the [Docker Compose compose file documentation](https://docs.docker.com/compose/compose-file/)
+so that you have a handle on how to setup the various configuration.
+
+Please see the [Outrigger Examples Repository](https://github.com/phase2/outrigger-examples) for 
+how to setup various technologies within Outrigger.  Specifically, the 
+[Drupal 8 example](https://github.com/phase2/outrigger-examples/tree/master/drupal8) covers all
+of the high points.
+
```

Comment:
Nice adding this page! I think I have some fragmentary bits that might be a good section or two to add once this gets in.